File: C++/First Steps in Coding - Basics of C++/Conditional Statements - Advanced/Personal Titles.cpp
#include <iostream>
#include <cstdlib>
#include <string>
using namespace std;
int main() {
	double age;
	string sex;
	cin >> age >> sex;
	if (sex == "m" && age >= 16) {
		cout << "Mr.";
	}
	else if (sex == "m" && age < 16) {
		cout << "Master";
	}
	else if (sex == "f" && age >= 16) {
		cout << "Ms.";
	}
	else if (sex == "f" && age < 16) {
		cout << "Miss";
	}
	return 0;
}

File: Java/Advanced Conditional Statements/Lab/04. Personal Titles/PersonalTitles.java
import java.util.Scanner;

public class PersonalTitles {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double age = Double.parseDouble(input.nextLine());
        String sex = input.nextLine();

        if(sex.equals("m")){
            if(age < 16){
                System.out.println("Master");
            } else {
                System.out.println("Mr.");
            }
        }else if(sex.equals("f")){
            if(age < 16){
                System.out.println("Miss");
            }else {
                System.out.println("Ms.");
            }
        }
    }
}
